uphold/vault-assist-tool
# Add support for RLUSD token vault

### Description
- Add support for RLUSD token vault

### Testing instructions
To start with testnet:
```
NET=development yarn dev
```

To start with mainnet:
```
yarn start
```

1. Start the web server the correct environment (above)
2. Open browser to `http://localhost:3000/`
3. Make sure you have an XRPL multisig created, and funded with RLUSD
5. Make sure that you have a second (destination) account funded with RLUSD 
6. In Vault Assist Tool, select `RLUSD` from the Asset type
7. Enter the vault address and continue
8. Enter the destination address of your secondary (destination) account that already has the token trust line
9. Confirm that the amounts and fees look correct.
10. Submit transaction, confirm "view transaction" opens the correct explorer link
11. Confirm no other issues with fees or amounts

### Reference
LABS-943
